lib/rest2.js

```diff
@@ -1340,6 +1340,58 @@ class RESTv2 {
       .then(_takeResNotify)
   }
 
+  /**
+   * Submit multiple orders.
+   *
+   * @param {order[]} orders - list of orders (can be object literal or Order class instance)
+   * @param {Function} [cb]  - callback
+   * @returns {Promise} p
+   */
+  submitOrderMulti (orders = [], cb) {
+    const payload = orders.map((order) => {
+      // generate new packet from instance of Order class to ensure consistent payload signature
+      const packet = (order instanceof Order ? order : new Order(order)).toNewOrderPacket()
+      if (this._affCode) {
+        packet.meta.aff_code = packet.meta.aff_code || this._affCode
+      }
+
+      return ['on', packet]
+    })
+
+    return this._makeAuthRequest('/auth/w/order/multi', { ops: payload }, cb)
+      .then(_takeResNotify)
+  }
+
+  /**
+   * Update multiple orders.
+   *
+   * @param {order[]} orders - list of orders (can be object literal or Order class instance)
+   * @param {Function} [cb]  - callback
+   * @returns {Promise} p
+   */
+  updateOrderMulti (orders = [], cb) {
```

Comment:
Same here: 
if (!Array.isArray(orders)) {
const e = new Error('orders should be an array')
return this._cb(e, null, cb)
}